internal/db/storage/blob/s3/writer/writer_test.go

```diff
@@ -132,54 +150,34 @@ func Test_writer_Open(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           ctx: nil,
-		       },
-		       fields: fields {
-		           eg: nil,
-		           service: nil,
-		           bucket: "",
-		           key: "",
-		           maxPartSize: 0,
-		           pw: nil,
-		           wg: sync.WaitGroup{},
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           ctx: nil,
-		           },
-		           fields: fields {
-		           eg: nil,
-		           service: nil,
-		           bucket: "",
-		           key: "",
-		           maxPartSize: 0,
-		           pw: nil,
-		           wg: sync.WaitGroup{},
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns nil when no error occurs",
```

Comment:
can you add a test case for error case?

Reply:
btw do you know why we use waitgroup here?
https://github.com/vdaas/vald/pull/672/files#diff-da7d1c78589548bbb90ba2560947237bR69

Reply:
please ignore this comment 
`btw do you know why we use waitgroup here?`